src/components/Shop/Navbar.tsx

```diff
@@ -12,7 +16,14 @@ const Navbar: FC = () => {
         />
       </div>
       <div>
-        <span>Ola Nordmann | (icon) xxxkr - xxx Ølcoins (icon)</span>
+        {user ? (
```

Comment:
This ternarary check is not necessary. If the user is not logged in, they will be routed to the login page. So this will never happen.

Reply:
Typescript throws an error if I dont have the ternarary check. 
![Capture](https://user-images.githubusercontent.com/32820275/98588472-0a839380-22cc-11eb-8c54-efcfe78638dc.PNG)